manifest.php

```diff
@@ -41,7 +41,7 @@
     'label'       => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license'     => 'GPL-2.0',
-    'version'     => '34.3.1',
+    'version'     => '34.3.2',
```

Comment:
Same remark, let's jump a feature version and enforce the dependency to `tao >= 38.5.0` and `taoTests` also

Reply:
ok